index.html

```diff
@@ -902,7 +902,7 @@ <h5 class="text-center">Environmental Map (DEQ)</h5>
         <h5 class="text-center">SGID Parcel Viewer (UGRC)</h5>
         <img alt="SGID Parcel Viewer (UGRC)" class="lozad website-thumb"
           src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
-          data-src="{% link images/parcels_utah_gov.png %}" />
+          data-src="{% link images/applications/parcels_utah_gov.png %}" />
       </a>
       <p class="muted">Zoom in to view parcel boundaries and basic attributes as assembled statewide by UGRC, in
```

Comment:
it'd be nice to add the plss app in here to have an even grid. I think we removed the crash app once upon a time and never replaced it.